aten/src/ATen/native/xnnpack/Engine.h

```diff
@@ -90,8 +90,7 @@ Tensor channel_shuffle(
 // Activations
 //
 bool use_hardswish(const Tensor& input);
-Tensor hardswish(const Tensor& input);
-Tensor& hardswish_(Tensor& input);
```

Comment:
Looks like there are some mobile tests at `vulkan_api_test.cpp` that expect to use this. Maybe it's worth refactoring them to use the out= variant? (Or worst case, adding this back)